nengo_spa/algebras/hrr_algebra.py

```diff
@@ -62,6 +67,45 @@ def is_valid_dimensionality(self, d):
         """
         return d > 0
 
+    def create_vector(self, d, properties, *, rng=None):
+        """Create a vector fulfilling given properties in the algebra.
+
+        Parameters
+        ----------
+        d : int
+            Vector dimensionality
+        properties : set of str
+            Definition of properties for the vector to fulfill. Valid set
+            elements are constants defined in `.HrrProperties`.
+        rng : numpy.random.RandomState, optional
+            The random number generator to use to create the vector.
+
+        Returns
+        -------
+        ndarray
+            Random vector with desired properties.
+        """
+        properties = set(properties)
+
+        if rng is None:
+            rng = np.random.RandomState()
+
+        v = rng.randn(d)
+        v /= np.linalg.norm(v)
+
+        if HrrProperties.POSITIVE in properties:
+            properties.remove(HrrProperties.POSITIVE)
+            v = self.bind(self.sign(v).to_vector(d), v)
```

Comment:
I think this can call out to `abs` since the default implementation for `abs` is exactly this?